File: L004/Patterns/pattern2.java
import java.util.Scanner;

public class pattern2 {

    public static Scanner sc = new Scanner(System.in);

    public static void pattern2_(int row) {
        int nst = row;
        for (int r = 1; r <= row; r++) {
            for (int cst = 1; cst <= nst; cst++) {
                System.out.print("*\t");
            }
            nst--;
            System.out.println();
        }
    }

    public static void main(String[] args) {

        int n = sc.nextInt();

        pattern2_(n);

    }
}

File: L004/Patterns/pattern2.cpp
#include<iostream>
using namespace std;


void pattern2_(int row){
    int nst=row;
    for(int r=1;r<=row;r++){
        for(int cst=1;cst<=nst;cst++){
            cout<<"*\t";
        }
        nst--;
        cout<<endl;
    }
}

int main(){

    int n;
    cin>>n;

    pattern2_(n);
    return 0;
}